utility/servicer/module.go

```diff
@@ -138,19 +235,86 @@ func (s servicer) validateRelayChainSupport(relayChain *coreTypes.Identifiable,
 	return nil
 }
 
-// TODO: implement
 // validateApplication makes sure the application has not received more relays than allocated in the current session.
-func (s servicer) validateApplication(meta *coreTypes.RelayMeta, session *coreTypes.Session) error {
-	/*
-		// if maxRelaysPerSession, overServiced := calculateAppSessionTokens(); overServiced {
-			return fmt.Errorf("application %s has exceeded its allocated relays %d for the session %d", meta.ApplicationPublicKey, maxRelaysPerSession)
-		}
-	*/
-	return nil
+func (s *servicer) validateApplication(session *coreTypes.Session) error {
```

Comment:
I feel like we shold rename `validateApplication` to `sholdMineRelay`. Optionally returning a boolean or just using a nil error to signify `yes`

Reply:
Thank you, fixed the name. Had to keep the return type as error: even though a boolean is preferable here, the `shouldMineRelay` function could run into errors: so just returning nil would indicate it is OK to mine.